corporate/lib/stripe.py

```diff
@@ -2498,6 +2498,57 @@ def update_license_ledger_for_automanaged_plan(
         # needs the updated plan for a correct LicenseLedger update.
         return plan
 
+    def migrate_customer_to_legacy_plan(
+        self,
+        renewal_date: datetime,
+        end_date: datetime,
+    ) -> None:
+        # Set stripe_customer_id to None to avoid customer being charged without a payment method.
```

Comment:
It might make sense to assert `self` is not a `RealmBillingSession`, but realistically that's not going to come up, so it'd just be for readability.

Reply:
Done.